x-pack/plugins/security_solution/public/flyout/entity_details/user_right/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const INSTALL_EA_INTEGRATIONS_HREF = `browse/security?q=entityanalytics`;
```

Comment:
can this be merged with the other constants in shared folder

Reply:
the reason I kept it separate is because this constant is used in a single place within the `user_right` folder